plugins/events/create.js

```diff
@@ -39,11 +39,23 @@ module.exports = () => ({
                                     pipelineId: j.pipelineId,
                                     startFrom: j.name,
                                     parentBuildId: b.parentBuildId,
-                                    parentEventId: b.eventId
+                                    parentEventId: b.eventId,
+                                    groupEventId: request.payload.groupEventId,
+                                    parentBuilds: b.parentBuilds
                                 };
 
-                                if (b.parentBuilds) {
-                                    restartConfig.parentBuildsInfo = b.parentBuilds;
+                                if (!restartConfig.parentBuilds) {
+                                    restartConfig.parentBuilds = request.payload.parentBuilds;
```

Comment:
does this require ui changes?

Reply:
should be the same logic as before, just refactored